src/xml_comment.cc

```diff
@@ -82,7 +82,7 @@ XmlComment::get_content() {
     return NanEscapeScope(ret_content);
   }
 
-  return NanEscapeScope(NanNew<v8::String>());
+  return NanEscapeScope(NanNew<v8::String>(""));
```

Comment:
The empty string constructor was accidentally omitted in the NanNew rewrite. It will be restored in the next patch release (next week). https://github.com/rvagg/nan/pull/237